packages/tracing-internal/src/browser/request.ts

```diff
@@ -195,9 +195,16 @@ function resourceTimingEntryToSpanData(resourceTiming: PerformanceResourceTiming
   }
   return [
     ...timingSpanData,
-    ['http.request.connect_start', (browserPerformanceTimeOrigin + resourceTiming.connectStart) / 1000],
-    ['http.request.request_start', (browserPerformanceTimeOrigin + resourceTiming.requestStart) / 1000],
-    ['http.request.response_start', (browserPerformanceTimeOrigin + resourceTiming.responseStart) / 1000],
+    ['http.request.redirect_start', resourceTiming.redirectStart],
+    ['http.request.fetch_start', resourceTiming.fetchStart],
+    ['http.request.domain_lookup_start', resourceTiming.domainLookupStart],
+    ['http.request.domain_lookup_end', resourceTiming.domainLookupEnd],
+    ['http.request.connect_start', resourceTiming.connectStart],
```

Comment:
Just to be clear, is it on purpose that we changed this from:

```js
(browserPerformanceTimeOrigin + resourceTiming.connectStart) / 1000
```

to just

```js
resourceTiming.connectStart
```

?

Reply:
I did intentionally do that, but I'm still trying to figure whether it makes more sense to align against browserTimingOrigin or just record the the values as is and change the representation internally. Mostly since I've seen these reported values be less than the wrapping span `startTime` so something isn't adding up.

Reply:
Actually after thinking about it some more, I'm changing it back. Absolute time makes the most sense I think it's up to the sdk to provide time normalized against `span` start/end time not a representation concern.